proto/autokitteh/sessions/v1/svc.proto

```diff
@@ -36,18 +36,13 @@ message ListRequest {
 
   bool count_only = 10;
 
-  // TODO: Pagination.
   // TODO: FieldMask.
 
-  int32 page_size = 20 [(buf.validate.field).cel = {
-    id: "session.list.page_size",
-    message: "Must be between 10 and 100",
-    expression: "this >= 10 && this <= 100"
-  }];
+  int32 page_size = 20; // Invalid values are fixed by the gRPC service.
```

Comment:
could u please elaborate when/where/how it fixed?

Reply:
could u plz add TODO in sessionsgrpcsvc to remove magic values to some config/consts?

Reply:
Configurability: services are always the deciders when it comes to pagination, based on our understanding of performance. This is not a setting that users should be able to control, so they're always hard-coded and not configurable.

As for consts - that's a good idea, but beyond the scope of this PR. I opened this ticket: https://linear.app/autokitteh/issue/ENG-1033/improvements-in-pagination-in-grpc-services